pom.xml

```diff
@@ -58,7 +58,7 @@
         <artifactId>appengine-maven-plugin</artifactId>
         <version>2.2.0</version>
         <configuration>
-          <deploy.projectId>CAPSTONE_PROJECT_ID</deploy.projectId>
+          <deploy.projectId>xinhaofighting-step-2020</deploy.projectId>
```

Comment:
Was this a temporary switch for testing purposes?

Reply:
Yes. I use my own porfolio for further deployment testing